lib/taiko.js

```diff
@@ -638,6 +638,19 @@ module.exports.deleteCookies = async (cookieName, options = {}) => {
   }
 };
 
+/**
+ * Resize the browser window
+ *
+ * @example
+ * await resizeWindow({width:600, height:800})
+ *
+ * @returns {Promise<void>}
+ */
+module.exports.resizeWindow = async (options = {}) => {
+  const [{ targetId }] = await targetHandler.getFirstAvailablePageTarget();
+  await browserHandler.setWindowBounds(targetId, options.height, options.width);
```

Comment:
Please add check for height and width parameter, both of them should be passed. Throw an error in this case.